qiskit_sphinx_theme/pytorch_base/static/css/theme.css

```diff
@@ -1,4 +1,12 @@
 @charset "UTF-8";
+
+/* Import IBM Plex Sans and IBM Plex Mono, at regular 400 and bold 700, italics for both. For
+IBM Plex Sans, also include font weights 500 and 600 (non-italics), which are used in table of
+contents. */
```

Comment:
I wasn't totally sure how to reason about this. For example:

* should Mono have 500 and 600?
* should Sans have italics for 500 and 600? Which seem to be used for table of contents

Generally, is it better to be "safe", or should we try to focus on better performance? How much do we care about performance and low-bandwidth with qiskit.org?